examples/00-EDB/10_GDS_workflow.py

```diff
@@ -0,0 +1,104 @@
+"""
+EDB: Edit Control File and import gds
+-------------------------------------
+This example shows how you can use PyAEDT to import a gds from an IC file.
+"""
+
+###############################################################################
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Perform required imports, which includes importing a section.
+
+import os
+import tempfile
+import pyaedt
+import shutil
+from pyaedt.edb_core.edb_data.control_file import ControlFile
+
+###############################################################################
+# Download file
+# ~~~~~~~~~~~~~
+# Download the AEDB file and copy it in the temporary folder.
+temppath = tempfile.gettempdir()
+local_path = pyaedt.downloads.download_file('gds')
+c_file_in = os.path.join(
+    local_path, "example_models", "cad", "GDS", "sky130_fictitious_dtc_example_control_no_map.xml"
+)
+c_map = os.path.join(local_path, "example_models", "cad", "GDS", "dummy_layermap.map")
+gds_in = os.path.join(local_path, "example_models", "cad", "GDS", "sky130_fictitious_dtc_example.gds")
+gds_out = os.path.join(temppath, "example.gds")
+shutil.copy2(gds_in,gds_out )
+###############################################################################
+# Control file
+# ~~~~~~~~~~~~
+# A Control file is an xml file which purpose if to provide additional
+# information during import phase. It can include, materials, stackup, setup, boundaries and settings.
+# In this example we will import an exising xml, integrate it with a layer mapping file of gds
+# and then adding setup and boundaries.
```

Comment:
```suggestion
# A control file is an XML file that provides additional information during the
# import phase. It can include materials, stackup, setup, boundaries, and settings.
# This example imports an existing XML file, integrates it with a layer mapping file of gds,
# and then adds the setup and boundaries.
```
If "gds" is an acronym or abbreviation, use all capital letters in the second to last line.